rules/falco_rules.yaml

```diff
@@ -307,13 +305,13 @@
 # sshd, sendmail-msp, sendmail attempt to setuid to root even when running as non-root. Excluding here to avoid meaningless FPs
 - rule: non_sudo_setuid
   desc: an attempt to change users by calling setuid. sudo/su are excluded. user "root" is also excluded, as setuid calls typically involve dropping privileges.
-  condition: evt.type=setuid and evt.dir=> and not user.name=root and not userexec_binaries and not proc.name in (sshd, sendmail-msp, sendmail)
+  condition: evt.type=setuid and evt.dir=> and not user.name=root and not proc.name in (userexec_binaries, sshd, sendmail-msp, sendmail)
```

Comment:
so a list `(a, b, c)` can contain a mix of lists and individual elements, and the result is a list where the included lists have been automatically expanded? e.g. `(a, b, [c, d, e])` is the same thing as `(a, b, d, c, e)`?